go.mod

```diff
@@ -0,0 +1,33 @@
+module test/v2
```

Comment:
Could we have those Go files in some tests/dev directory? They may be confusing for anyone visiting the repository.

Reply:
It has to be parent dir of the go source file, what folder structure would u suggest?

Reply:
How about we put all the go tests in `go_compat/tests/` and those mod files in `go_compat/`? Would that work?

Reply:
That works but making it a bit hard to locate the file from rust code. Maybe worth adding some test utilities

Reply:
Fixed

Reply:
rock-solid, thanks!